health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/components/UploadDataMapping.js

```
@@ -552,26 +556,33 @@
-        customStyles={tableCustomStyle}
-        // paginationTotalRows={totalRows}
-        // onChangePage={handlePaginationChange}
-        // onChangeRowsPerPage={handleRowsPerPageChange}
-        // paginationPerPage={rowsPerPage}
-        paginationTotalRows={state.totalRows}
-        onChangePage={handlePageChange}
-        onChangeRowsPerPage={handleRowsPerPageChange}
-        paginationRowsPerPageOptions={[5, 10, 15, 20]}
+        shapeOnOff={true}
       />
+      <div></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)